### DISCUSSION TITLE
comma in url broke httpx

### DISCUSSION THREAD

**Discussion by molitona** [projectdiscovery/httpx#1001] [Q&A]
hi comma in url broke httpx, if https://xx.xx?perms=X,Y,Z is used httpx will think X,Y,Z are paths but they're part of url as options to param perms.

**Comment by olearycrew** [comment]
Commas are not a part of the URI RFC - see this Stack Overflow answer for more details: https://stackoverflow.com/questions/198606/can-i-use-commas-in-a-url

Could you URL encode those values instead?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
